src/event.re

```diff
@@ -0,0 +1,49 @@
+type event = {id: string, title: string, description: string, time: float};
+
+let component = ReasonReact.statelessComponent "Event";
+
+type props = {event: event};
+
+module Decode = {
+  let event j =>
+    Json.Decode.{
+       id: j |> field "id" string,
+       title: j |> field "name" string,
+       description: j |> field "description" string,
+       time: j |> field "time" float
+     };
+
+  let root json => {
+    Json.Decode.(array event json);
```

Comment:
Yes! I found it in the bs-json examples, there are array and list types that take a type as an argument!